scripts/test/lld.py

```diff
@@ -75,5 +50,4 @@ def test_wasm_emscripten_finalize():
 
 
 if __name__ == '__main__':
-  test_wasm_link_metadata()
   test_wasm_emscripten_finalize()
```

Comment:
add `sys.exit(0)` or change to `sys.exit(test_wasm_emscripten_finalize())`?

Reply:
None of the other check.py sub-scripts do so I'm inclined to leave it as-is.